packages/fuel-gauge/src/transaction-summary.test.ts

```diff
@@ -314,6 +314,131 @@ describe('TransactionSummary', () => {
       });
     });
 
+    it('should ensure transfer operation is assembled (UNLOCKING PREDICATE)', async () => {
```

Comment:
This test I was just playing around with an additional use case and exploring what should be display.

I'm happy with this case.